packages/astro/e2e/errors.test.js

```diff
@@ -131,4 +131,12 @@ test.describe('Error display', () => {
 		const message = (await getErrorOverlayContent(page)).message;
 		expect(message).toMatch('The operation was aborted due to timeout');
 	});
+
+	test('properly highlight the line with the error zzz', async ({ page, astro }) => {
```

Comment:
```suggestion
	test('properly highlight the line with the error', async ({ page, astro }) => {
```

Reply:
I left it in there on purpose to make sure this PR was properly reviewed. (I did not)